src/Api/Health/HealthCheckServices.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Bit.Core.Settings;
+
+namespace Bit.Api.Health;
```

Comment:
I think we want to move this into core so we can have this health check on all services, especially identity. Identity doesn't need to call itself though so we should expose the builder outside this method and non-identity projects and add that one themself.

Reply:
We should move it to `SharedWeb` instead of Core since core ideally shouldn't know about asp.net stuff (Pipe dream I know). And also figure out which health checks needs to be run on each project since that will vary widely. Meaning we might need custom healths per service.

Reply:
Didn't know others had that pipe dream, `SharedWeb` it is!